client/deposits/details/index.tsx

```diff
@@ -160,20 +169,61 @@ export const DepositOverview = ( {
 	);
 };
 
-export const DepositDetails = ( {
+interface DepositDetailsProps {
+	query: {
+		id: string;
+	};
+}
+
+export const DepositDetails: React.FC< DepositDetailsProps > = ( {
 	query: { id: depositId },
-}: {
-	query: { id: string };
-} ): JSX.Element => (
-	<Page>
-		<TestModeNotice topic={ topics.depositDetails } />
-		<ErrorBoundary>
-			<DepositOverview depositId={ depositId } />
-		</ErrorBoundary>
-		<ErrorBoundary>
-			<TransactionsList depositId={ depositId } />
-		</ErrorBoundary>
-	</Page>
-);
+} ) => {
+	const { deposit = {} as CachedDeposit, isLoading } = useDeposit(
+		depositId
+	);
```

Comment:
This hook has been hoisted from `<DepositOverview />` to the parent component `<DepositDetails />` so it can access the `deposit` to determine if it is instant or scheduled.